app/src/main/java/berlin/eloquent/eloquentandroid/main/repository/RecorderRepository.kt

```diff
@@ -49,4 +53,10 @@ class RecorderRepository @Inject constructor(
         }
     }
 
+    override suspend fun getAnalysis(): Response<DataModel> {
+        return withContext(ioDispatcher) {
+            return@withContext service.getAnalysis()
+        }
+    }
+
 }
```

Comment:
```suggestion
}

```

Reply:
new lines should be at the end of each file

Reply:
For a specific reason or just for the looks ?